src/dmtcp_coordinator.cpp

```diff
@@ -467,6 +468,7 @@ void DmtcpCoordinator::updateMinimumState(WorkerState oldState)
     if (exitAfterCkpt || exitAfterCkptOnce) {
       JNOTE("Checkpoint Done. Killing all peers.");
       broadcastMessage(DMT_KILL_PEER);
+      JTIMER_STOP ( checkpoint );
```

Comment:
I followed the convention from earlier, where we'd stop the checkpoint timer after broadcasting messages to the peers. See: https://github.com/dmtcp/dmtcp/blob/2.5/src/dmtcp_coordinator.cpp#L491